### ISSUE TITLE
[Feature request] Provide the trainer of dvae and hifigan decoder in xtts2 model.

### ISSUE THREAD

**Issue by Georgehappy1** [coqui-ai/TTS#3267]
**🚀 Feature Description**
Thanks for your great work of xtts2. Recently you have open sourced the trainer of gpt2 of xtts2. I wonder if you have plan to also provide the trainer of dvae and hifigan decoder:?

**Solution**

**Alternative Solutions**

**Additional context**

**Comment by Liujingxiu23** [comment]
@erogol  hi,   I did not understand how to train the HifiDecoder model.
In inference of xtts2, the logits of audio-tokens are used as input of HifiDecoer? Is my understanding right?
But in trainning of HifiDecoder model, there is no logits? Then how to train the model?
I do not ask for code, just want to know the main process .

**Comment by tuanh123789** [comment]
The code for training hifigan available here: https://github.com/tuanh123789/Train_Hifigan_XTTS
